inc/container.class.php

```diff
@@ -1417,6 +1417,17 @@ public static function validateValues($data, $itemtype, $massiveaction)
         $status_overrides = array_key_exists($status_field_name, $data) && $data[$status_field_name] !== null
             ? PluginFieldsStatusOverride::getOverridesForItemtypeAndStatus($container->getID(), $itemtype, $data[$status_field_name])
             : [];
+
+        // If no overrides are loaded, it may be because by the container if it's type is 'tab'
```

Comment:
It may also be because there is no overrides for the given status. We should adapt logic on container type.
```php
$status_value = null;
if ($container->fields['type'] === 'dom') {
    $status_value = $data[$status_field_name] ?? null;
} else {
    $status_value = $relatedItem->fields[$status_field_name] ?? null;
}
$status_overrides = $status_value !== null
            ? PluginFieldsStatusOverride::getOverridesForItemtypeAndStatus($container->getID(), $itemtype, $status_value)
            : [];
```

Reply:
done !